package.json

```diff
@@ -87,6 +87,7 @@
     "karma-mocha": "^0.2.2",
     "karma-mocha-reporter": "^2.0.0",
     "karma-phantomjs-launcher": "^1.0.0",
+    "karma-phantomjs-shim": "github:technologyadvice/karma-phantomjs-shim",
```

Comment:
Can go back to main branch once String.repeat and String.includes are merged:

https://github.com/tschaub/karma-phantomjs-shim/pull/7
https://github.com/tschaub/karma-phantomjs-shim/pull/8